wire-ios-data-model/Source/MLS/MLSGroup.swift

```diff
@@ -56,7 +56,6 @@ public class MLSGroup: ZMManagedObject {
         }
     }
 
-    @objc
```

Comment:
should it go to next line?

Reply:
I think it's not even needed in this case that's why it's being removed.